packages/core/lib/engine_http.js

```diff
@@ -399,12 +401,22 @@ HttpEngine.prototype.step = function step(requestSpec, ee, opts) {
           requestParams.body = _.reduce(
             requestParams.formData,
             function (acc, v, k) {
-              // acc[k] = template(v, context);
-              acc.append(k, template(v, context));
+              let V = template(v, context);
+              if (V && _.isPlainObject(V) && V.fromFile) {
+                V = fs.createReadStream(
+                  path.resolve(process.cwd(), V.fromFile)
```

Comment:
This needs to be resolved relative to the test script itself (same way we're doing it for `processor` or `payload` entries for example). Otherwise the test script will work only when the CLI is run from a specific directory.

Reply:
Addressed this by making path to scenarios available as test level variable `$scenarioFile` as discussed. Commit https://github.com/artilleryio/artillery/pull/3232/commits/4dd0f57a4af69e5be938320e647688cd3974b17c